src/devices/Is31fl3730/BreakoutPair5x7.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Threading;
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Represents an IS31FL3731 LED Matrix driver
+    /// </summary>
+    // Datasheet: https://cdn-shop.adafruit.com/product-files/3017/31FL3730.pdf
+    // Product: https://shop.pimoroni.com/products/microdot-phat
+    // Product: https://shop.pimoroni.com/products/led-dot-matrix-breakout
+    // Related repo: https://github.com/pimoroni/microdot-phat
+    public class BreakoutPair5x7 : IDisposable
+    {
+        private readonly Matrix3730 _matrixOne;
+        private readonly Matrix3730 _matrixTwo;
+        private readonly Matrix3730[] _pair = new Matrix3730[2];
+        private Is31fl3730 _is31fl3730;
+        private I2cDevice? _i2cDevice;
+
+        /// <summary>
+        /// Initialize IS31FL3730 device
+        /// </summary>
+        /// <param name="i2cDevice">The <see cref="System.Device.I2c.I2cDevice"/> to create with.</param>
+        public BreakoutPair5x7(I2cDevice? i2cDevice = null)
+        {
+            _i2cDevice = i2cDevice is not null ? i2cDevice : I2cDevice.Create(new(1, Is31fl3730.DefaultI2cAddress));
```

Comment:
We're not using this pattern and always leave the creation of the I2cDevice to the caller. Reason is because bus may be different, ownership as well and finally adaptors like FT4222 or Arduino. So don't pass the null as default and raise and exception if null.